Documentation/building/unix-instructions.md

```diff
@@ -112,21 +112,27 @@ ln -s /usr/local/opt/openssl/lib/pkgconfig/openssl.pc /usr/local/lib/pkgconfig/
 
 Alternatively, to avoid modifying /usr/local/ you can invoke cmake with the `OPENSSL_ROOT_DIR` env var set. The value to be passed in the directory where openssl is installed. Use `brew info openssl` to determine it. For example:
```

Comment:
If the argument for not invoking cmake below is step 4 (above) said to use `./build.sh`, presumably this reference to cmake should be changed to `./build.sh`?